README.md

```
@@ -1,55 +1,31 @@
+A monorepo containing different tools for rust development.
 
 This monorepo contains the following packages:
 
-### [@webway/data-parser](./packages/webway-parser)
-High-performance data parser built with Rust for processing binary data formats.
-
-**Features:**
-- Binary data parsing
-- Memory-efficient processing
-- TypeScript bindings
+### [@void-ark/panamax](./packages/panamax)
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)